extensions/mssql/src/objectManagement/localizedConstants.ts

```diff
@@ -260,6 +260,7 @@ export const StorageSpaceUsageInMBText = localize('objectManagement.storageSpace
 export const VersionText = localize('objectManagement.versionText', "Version");
 export const minServerMemoryText = localize('objectManagement.minServerMemoryText', "Minimum Server Memory (MB)");
 export const maxServerMemoryText = localize('objectManagement.maxServerMemoryText', "Maximum Server Memory (MB)");
+export const serverMemoryMaxLowerThanMinInputError: string = localize('objectManagement.serverMemoryMaxLowerThanMinInputError', "Maximum server memory cannot be lower than server minimum memory")
```

Comment:
```suggestion
export const serverMemoryMaxLowerThanMinInputError: string = localize('objectManagement.serverMemoryMaxLowerThanMinInputError', "Maximum server memory cannot be lower than minimum server memory")
```
Small change for consistent phrasing in the message.